recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "wntr" %}
-{% set version = "v0.4.2" %}
+{% set version = "v0.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/USEPA/WNTR/archive/0.4.2.zip
-  sha256: c76fb0f1b1c40c1df3122aa575de08f75a62497cc522a081ecb6929ffd8069e0
+  url: https://github.com/USEPA/WNTR/archive/0.5.0.zip
+  sha256: 21bb1fde619094e550e9fec338feb3f78268596f8a9db7cace69aa4c216ca216
 
 build:
-  number: 0
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
I think we need to now specify that the extensions need built. We should be able to do it with the environment variable.

Reply:
By setting BUILD_WNTR_EXTENSIONS='true'?  where should that be defined?

Reply:
Yes. I'm not sure yet. I'm looking.

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#use-environment-variables
```
build:
  number: 0
  script_env:
    - BUILD_WNTR_EXTENSIONS='true'
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Does that look right?

Reply:
That does look right. I can check the actions to be sure.

Reply:
Looks like the env variable was set, but the build still fails.